psl-devices

```diff
@@ -7,8 +7,8 @@
 0x3003 AlphaDataKU60 Xilinx
 0x0604 AlphaData7v3 Xilinx
 0x0605 AlphaDataKU60 Xilinx
-0x0606 AlphaData Xilinx
-0x0607 AlphaData Xilinx
+0x0606 SemtpianNSA121A Xilinx
```

Comment:
These card types do not match the information I have for PSL External Releases. Are these subIDs coming from IBM?